ports/comms-ublox/portfile.cmake

```
@@ -3,28 +3,22 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO commschamp/cc.ublox.generated
-    REF v0.20.2
-    SHA512 5672d964ea3e505837e44a5fd928069a219a5731764cb54bfe8609e39c6c6dd0059660bcde317c6c60cd1bd8d1f7942d2faa022095bf651817568291bc6a7569
+    REF v1.0
+    SHA512 0c487d9409c2f2818024f6232832762527250c3563a5eb5c639ad49943931ceb24616db2432bcd752d1a84820ec5349522510dcd202508641d3f29aef41ca1e5
     HEAD_REF master
-    PATCHES
-        fix-comms.patch
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    PREFER_NINJA
     OPTIONS
-        -DOPT_BUILD_TEST=OFF
-        -DOPT_BUILD_PLUGIN=OFF
-        -DOPT_NO_COMMS=ON
-        -DOPT_EXTERNALS_UPDATE_DISCONNECTED=ON
+        -DOPT_REQUIRE_COMMS_LIB=ON

```

```suggestion
        -DOPT_REQUIRE_COMMS_LIB=ON
        -DBUILD_TESTING=OFF
```
Upstream doesn't handle `CC_COMMS_BUILD_UNIT_TESTS` properly. See https://github.com/commschamp/comms/blob/v4.0/test/CMakeLists.txt